playbooks/roles/sit.gpfs/tasks/setup/redhat.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Install required packages
+  yum:
+    name:
+      - ansible-core
+      - ansible-collection-ansible-posix
+      - ansible-collection-community-general
+      - unzip
+    state: latest
+
+- name: Install required ansible collections
+  command: ansible-galaxy collection install -p /usr/share/ansible/collections community.crypto
```

Comment:
Doesn't it work with the default installation path?

Reply:
By default it installs collections into the user's private directory (_~/.ansible/collections/ansible_collections_)

Reply:
I can confirm that `community.crypto` ansible collection is available for install from epel which should be enabled in _setup_ machine. May be we could try installing **_ansible-collection-community-crypto_** instead?

Reply:
The package is not available for CentOS 8.

Reply:
Is it OK to add `become: no` to avoid specifying the installation path? See https://github.com/samba-in-kubernetes/sit-environment/blob/main/playbooks/roles/sit.glusterfs/tasks/setup/centos.yml#L16

Reply:
With latest changes, playbooks on setup0 node are executed as root now, so I'm not sure if this is still relevant. I'll try without the explicit path.